modules/stochastic_tools/include/state_sim/model/StateSimModel.h

```diff
@@ -0,0 +1,58 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef STATESIMMODEL_H
+#define STATESIMMODEL_H
+
+#include <string>
+#include <vector>
+#include "StateSimEvTimeBased.h"
+#include "StateSimEvConditionBased.h"
+#include "StateExternalCouplingUO.h"
+#include "StateSimAllEvents.h"
+#include "StateSimAllStates.h"
+#include "StateSimAllDiagrams.h"
+#include "StateSimAllVariables.h"
+#include "StateSimAllActions.h"
+
+/**
+ Object to load and run and manage a state simulation model
+ */
+class StateSimModel
+{
+public:
+  StateSimAllEvents time_events;
+  StateSimAllEvents condition_events;
+  StateSimAllStates states;
+  StateSimAllDiagrams diagrams;
+  StateSimAllVariables variables;
+  StateSimAllActions actions;
```

Comment:
You'll have to include each header that has a member variable like this. Again, we need leading underscores everywhere.